docs/asyncapi.yaml

```diff
@@ -0,0 +1,126 @@
+asyncapi: 3.0.0
+info:
+  title: Single front door user communications
+  version: 1.0.0
+  description: Publish communications to users via the Single Front Door
+
+channels:
+  fcp-fd-comms:
+    address: fcp-fd-comms-request
+    description: |
+      Accepts requests for communication notifications via Azure service bus.
+    messages:
+      notifyMessage:
+        $ref: '#/components/messages/notifyMessage'
+
+operations:
+  SourceSystem/fcp-fd-comms.subscribe:
+    action: receive
+    channel:
+      $ref: '#/channels/fcp-fd-comms'
+    
+
+components:
+  messages:
+    notifyMessage:
```

Comment:
I think our language is a bit confused.  We're using comms elsewhere, so could rename these `commsMessage` as opposed to `notifyMessage`?

Reply:
Was looking at the destination system, bus since we are moving away from being tightly coupled with gov.notify, I agree this needs to be changed